libs/execution_context/ebpf_general_helpers.c

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief Prototypes for general helper function implemented by eBPF core Execution Context.
```

Comment:
FYI regarding "general"... https://github.com/iovisor/bcc/blob/master/docs/kernel-versions.md#program-types calls them "base" functions.  Maybe state this in the comment here?